metricbeat/module/kubernetes/apiserver/_meta/docs.asciidoc

```diff
@@ -0,0 +1 @@
+This is the `apiserver` metricset of the Kubernetes module.
```

Comment:
@exekias I wonder if you left this out on purpose?

Reply:
probably not, :+1: to having it